src/components/button/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+import './index.less'
+export default class Button extends Component {
+    constructor() {
+        super()
+        this.classButton = this.classButton.bind(this)
+    }
+    classButton() {
```

Comment:
we should use this library [classnames](https://github.com/JedWatson/classnames) in the future